### PR TITLE
Fix run & quotation marks

### PR DESCRIPTION
**Pull request by bremoran**
Use updated suit-tool to replace "run" with "invoke" and to replace single quotes with double quotes in strings in CBOR diagnostic output

### CODE REVIEW

**Comment by kentakayama** [comment]
Every examples in diagnostic notation format can be converted to the same hex binary.

File: draft-ietf-suit-manifest.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ v: 3
 
 title: A Concise Binary Object Representation (CBOR)-based Serialization Format for the Software Updates for Internet of Things (SUIT) Manifest
 abbrev: CBOR-based SUIT Manifest
-docname: draft-ietf-suit-manifest-30
+docname: draft-ietf-suit-manifest-31
```
Comment (henkbirkholz) [inline draft-ietf-suit-manifest.md:6]:
```suggestion
docname: draft-ietf-suit-manifest-32
```

**Review by henkbirkholz — APPROVED** [review]
lgtm, but I messed up -31, so this will be -32

**Comment by henkbirkholz** [comment]
Wait,

sourcecode-types cddl and cbor-diag should get sourceoce-name lables

I.e.,
{: sourcecode-name="draft-ietf-suit-manifest.cddl"}
for the full cddl and
{: sourcecode-name="examples/example0.txt}
for example zero, one, etc.

For later automated checking and use of examples as well as cddl.

**Comment by bremoran** [comment]
@henkbirkholz please check out this update. I think it does the right thing.

**Pull request merged**